doc/OutlierFiltersFamilies.md

```diff
@@ -0,0 +1,377 @@
+
+Equations:
+![eq sparse](images/eq_sparse.JPG)
+
+Plots:
+![sparseICP fourk](images/sparseicp_fourk.png)
+
+Original reference:
+S.Bouaziz, A.Tagliasacchi, and M.Pauly, "Sparse iterative closest point", Eurographics Symposium on Geometry Processing, vol.32, no.5, pp. 113-123, 2013.
+
+#### EM-ICP <a name="emicphead"></a>
+
+The EM-ICP algorithm presents an effect on inliers similar to the step function. This is due it being based on L1. The effect on outliers is the same as the functions in the linear or sin families. Since the outlier filtering method is a trimmed version on L1 and it shares behavior with the step and linear families, it is placed in the other family.
+
+Equations:
+![eq emicp](images/eq_emicp.JPG)
+
+Plots:
+![emicp nok](images/emicp_nok.png)
+
+Original reference:
+S.Granger, and P.Xavier, "Multi-scale EM-ICP: A Fast and Robust Approach for Surface Registration", Proceedings of the European Conference on Computer Vision, pp. 418-432.
+
+#### RANSAC <a name="ransachead"></a>
+
+The RANSAC outlier filtering algorithm or RANdom SAmple Consensus relies on a voting system that uses inliers. The first step of the algorithm is to randomly sample the cloud of points. Then a model is applied depending on the problem. This model is applied to the sampled points. After the model is computed the algorithm counts the number of values contained within a [-k,k] range from the model. These values are considered as inliers. The model with the highest number of inliers is considered as the result of the algorithm. The RANSAC algorithm uses three parameters:
+* (s): the number of sampled points
+* (c): the number of computations
+* (k): the range containing the inliers
+Since the algorithm is based on random sampling, the results can be quite independent of k as shown with the plots. Indeed, for the same amount of points and the same number of computations the result for k = 0.5 ended up being poorer than the example using k = 1.5. This can be explained by the fact that (s) is the predominant tuning parameter for this algorithm.
+
+Plots:
+![ransac threek](images/ransac_threek.png)
+
+Original reference:
+M.A.Fischler and R.C. Bolles, "Random sample consensus: a paradigm for the model fitting with applications to image analysis and automated cartography", Communications of the ACM, vol.24, no.6, pp.381-395, 1981.
+
+Relative motion threshold (RMT):
+
+The relative motion threshold (RMT) uses a threshold based on translation between two transforms. In our simplified case this motion threshold was the mean of the absolute values of the translation on the y axis between two iterations.
+
+Equations:
+![eq rmt](images/eq_rmt.JPG)
+
+Plots:
+![rmt nok](images/rmt_nok.png)
```

Comment:
it is not clear what is going on in the graph

Reply:
Tried to re-explain it in the text of the RMT outlier filter